faq.html

```diff
@@ -3,9 +3,40 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./css/FAQ.css">
   <title>FAQ</title>
 </head>
 <body>
 
+	<h1> FAQ </h1>
+
+	<h2 class="headingalignment">About RDS </h2> 
+	<section class="sectionalignment">
+			<h3>Question 1 - What is RDS? </h3>
+			<h3>Answer - </h4>
+			<h3>Question 2 - How do I contribute to RDS? </h3>
+			<h3>Answer : As of now we're having 4 projects. Every project and the corresponding tech stack is mentioned below:
+			<ul>
+				<li>Welcome : HTML, CSS, Vanilla Javascript</li>
+				<li>Members : React</li>
+				<li>Events: React</li>
+				<li>Main App : Ember</li>
+			</ul>
+			 
+			       
+
+			</h3>
+
+	</section>
```

Comment:
Indentation is all spaced up. Let's fix that please

Reply:
Fixed.